### PR TITLE
HOTFIX: close iterator to avoid resource leak

### PR DESCRIPTION
**Pull request by mjsax**

### CODE REVIEW

**Review by lihaosky — APPROVED** [review]
LGTM!

**Review by bbejeck — APPROVED** [review]
LGTM

File: streams/src/main/java/org/apache/kafka/streams/kstream/internals/KStreamSessionWindowAggregate.java [hunk 1]
```diff
@@ -264,23 +264,25 @@ private void fetchAndEmit(final Record<KIn, VIn> record,
...
+                    final KeyValue<Windowed<KIn>, VAgg> kv = windowToEmit.next();
+
+                    tupleForwarder.maybeForward(
+                        record.withKey(kv.key)
+                            .withValue(new Change<>(kv.value, null))
+                            // set the timestamp as the window end timestamp
+                            .withTimestamp(kv.key.window().end())
+                            .withHeaders(record.headers()));
+                }
+          }
```
Comment (lihaosky) [inline streams/src/main/java/org/apache/kafka/streams/kstream/internals/KStreamSessionWindowAggregate.java]: Is this indentation off?

**Comment by bbejeck** [comment]
@mjsax it looks like the build is failing with a checkstyle error though maybe it's related to the indentation that @lihaosky pointed out.

**Comment by mjsax** [comment]
Merged to `trunk` and cherry-picked to `3.6` and `3.5` branches.

**Pull request merged**